system/core/base.php

```diff
@@ -51,14 +51,22 @@
 		never instantiated, it just collects static variables and methods.
 	*/
 	class Hypha {
+		public static $src;
 		public static $data;
 	}
 
+	Hypha::$src = new StdClass();
 	Hypha::$data = new StdClass();
-	Hypha::$data->css = new HyphaFile('data/hypha.css');
-	Hypha::$data->html = new HyphaFile('data/hypha.html');
-	Hypha::$data->digest = new HyphaFile('data/digest');
-	Hypha::$data->stats = new HyphaFile('data/hypha.stats');
+
+	Hypha::$src->css = 'data/hypha.css';
+	Hypha::$src->html = 'data/hypha.html';
+	Hypha::$src->digest = 'data/digest';
+	Hypha::$src->stats = 'data/hypha.stats';
+
+	Hypha::$data->css = new HyphaFile(Hypha::$src->css);
+	Hypha::$data->html = new HyphaFile(Hypha::$src->html);
+	Hypha::$data->digest = new HyphaFile(Hypha::$src->digest);
+	Hypha::$data->stats = new HyphaFile(Hypha::$src->stats);
```

Comment:
I second you on having no git-tracked files in data. Maybe the default theme should reside somewhere in core and be not-editable. 

Yes, in due time there should be a mechanism to copy the default theme to the theme dir and work from there. But since this partial solution already would lighten the work for working on themes, I would propose this go ahead.